eth2deposit/deposit.py

```diff
@@ -51,6 +55,38 @@ def check_python_version() -> None:
         sys.exit()
 
 
+def get_password(text: str) -> str:
+    return click.prompt(text, hide_input=True, show_default=False, type=str)
+
+
+def validate_password(cts: click.Context, param: Any, password: str) -> str:
+    is_valid_password = False
+
+    # The given password has passed confirmation
+    try:
+        validate_password_strength(password)
+    except ValidationError as e:
+        click.echo(f'Error: {e} Please retype again.')
```

Comment:
please retype again.
```suggestion
        click.echo(f'Error: {e} Please retype.')
```